### DISCUSSION TITLE
Testing Storybook + Vercel Preview + Github Actions

### DISCUSSION THREAD

**Discussion by steveg152** [storybookjs/storybook#27130] [Help]
### Summary

I am having an issue with running my tests from Github actions against my vercel deployment.  My preview is being successfully deployed and is accessible, but Storybook is still telling me that it is not available.  Here is a link to the failing action.  I am just using the same action as described in the docs.

https://github.com/steveg152/design-monorepo/actions/runs/9082242468/job/24958090975

**Comment by steveg152** [comment]
It seems this is just a basic authentication issue, the preview deployments require authentication.  Not sure what other ways there are to do this aside from paying Vercel for the extra deployment protection options so we can allow GitHub.

**Comment by feng-derivio** [comment]
There's an option on vercel side that allows public visit.
<img width="1637" alt="image" src="https://github.com/user-attachments/assets/e045f598-d62d-42f8-a0c9-42533299c5a0">

**Reply by feng-derivio** [reply]
Additionally, they allow ci/cd tools to bypass protection.
<img width="1458" alt="image" src="https://github.com/user-attachments/assets/1d43f43a-2e5a-4e97-a890-cc3f003da33a">
